### ISSUE TITLE
Accessibility - Main goods List Screen - Poor Color Contrast

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor-Android#15]
It's difficult to tell that there are three tabs on the Android application because of poor color contrast and no outline like on the iOS application.

**Comment by aquilent-dgemoets** [comment]
See discussion for #7.

**Comment by jfc3-dol** [comment]
It doesn't matter if these are standard Android elements if the color contrast is poor it makes it difficult for people with visual issues to read the text, which is what you want people to do and it only is worse if a person is viewing information on phone or tablet is bright sunlight. It's difficult to tell which tab is active.

**Comment by jfc3-dol** [comment]
The light blue text on the dark blue background for the tabs doesn't have enough color contrast.

**Comment by jfc3-dol** [comment]
The color contrast issues still persists in this version of the application.

**Comment by carolynhuang** [comment]
How do you suggest resolving this?  It seems like it would have substantial implications on the appearance of the app.

**Comment by jfc3-dol** [comment]
It doesn't matter if these are standard Android elements if the color contrast is poor it makes it difficult for people with visual issues to tell where one item starts and stops. You want people to use your application. It only is worse if a person is viewing information on phone or tablet is bright sunlight. It's difficult to tell which tab is active.

**Comment by jfc3-dol** [comment]
A good color would be @3d3d3d which is charcoal gray.

**Comment by jfc3-dol** [comment]
Thanks, for using the color #3d3d3d as a separator between each good so people can quickly tell them apart.

Closing issue since it's been corrected.
